web/docs/language/features.md

```diff
@@ -1171,15 +1171,19 @@ entity SocialLogin {=psl
   @@unique([provider, providerId, userId])
 psl=}
 ```
-
+:::caution
 Be sure to include an `externalAuthEntity` in your `auth` declaration, as [described here](features#externalauthentity). Note that the same `externalAuthEntity` can be used across different social login providers (e.g., both GitHub and Google can use the same entity).
+:::
 
-:::info
-Wasp assigns generated values to the `username` and `password` fields of the `userEntity` by default, so make sure to include it them your `userEntity` declaration even if you're only using a Social Login provider.
-
-If you require custom configuration setup or user entity field assignment, you can [override the defaults](features#google-overrides).
+:::info Default User Fields
+When a user signs in for the first time, Wasp assigns generated values to the `username` and `password` fields of the `userEntity` by default (e.g. `username: nice-blue-horse-14357`), so make sure to include these in your `userEntity` declaration even if you're only using a Social Login provider. If you'd like to change this behavior, these values can be overridden as described below.
 :::
 
+:::tip UI Helpers and Overrides
+Wasp provides you with a few UI helper components and functions to make implementing social login easier. See [here](features#ui-helpers-for-social-login-providers) for more details.
+
+It is also posslbe to [override the default](features#overrides-for-social-login-providers) login behaviors that Wasp provides for you. This allows you to create custom setups, such as allowing Users to define a username rather than the default random username assigned by Wasp on initial Login.
+:::
 
 #### Google
```

Comment:
Maybe make it a bigger title with `###` and maybe link to the guide below (e.g. "check out our guide on setting up Google auth")

<img width="560" alt="Screenshot 2023-01-30 at 16 10 22" src="https://user-images.githubusercontent.com/2223680/215515449-4944c42d-8820-4e59-b271-54add5a396ab.png">

Reply:
ok. I made some more improvements and changes to remove redundant information